Resources/Public/jsDomainModeling/wireit/js/WiringEditor.js

```diff
@@ -658,25 +659,30 @@
 		 * @method updateLoadPanelList
 		 */
 		updateLoadPanelList: function() {
-			var list = WireIt.cn("ul");
-			if (lang.isArray(this.pipes)) {
+		  var list;
+			if (lang.isArray(this.pipes) && this.pipes.length > 0) {
+        list = WireIt.cn("ul");
 				for (var i = 0; i < this.pipes.length; i++) {
 					var module = this.pipes[i];
 
 					this.pipesByName[module.name] = module;
 
-					var li = WireIt.cn('li', null, {cursor: 'pointer'}, module.name);
+					var li = WireIt.cn('li', null, {cursor: 'pointer'}, module.name + ' (stored in <code>' + module.storagePath + '</code>)');
+					li.dataset.name = module.name;
 					Event.addListener(li, 'click', function(e, args) {
 						try {
-							this.loadPipe(Event.getTarget(e).innerHTML);
+							this.loadPipe(Event.getTarget(e).dataset.name);
 						}
 						catch(ex) {
 							console.log(ex);
 						}
 					}, this, true);
 					list.appendChild(li);
 				}
-			}
+			} else {
+			  list = document.createElement('div');
+			  list.innerText = 'No extensions found...';
+      }
```

Comment:
Indentation is a bit off here

Reply:
EditorConfig was different to actual style used in this file...

Reply:
Good point. I updated the editorconfig some time ago, but we did not adjust all files.